ragaai_catalyst/tracers/agentic_tracing/tracers/llm_tracer.py

```
@@ -447,20 +449,20 @@ def wrapper(wrapped, instance, args, kwargs):
-        parameters={},
+            self,
+            component_id,
+            hash_id,
+            name,
+            llm_type,
+            version,
+            memory_used,
+            start_time,
+            input_data,
+            output_data,
+            cost={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)